be/src/exprs/runtime_filter.h

```diff
@@ -429,9 +458,11 @@ class RuntimeBloomFilter final : public JoinRuntimeFilter {
 
     void evaluate(Column* input_column, RunningContext* ctx) const override {
         if (_num_hash_partitions != 0) {
-            return _t_evaluate<true>(input_column, ctx);
+            return _hash_partition_bf[0].can_use() ? _t_evaluate<true, true>(input_column, ctx)
```

Comment:
when _hash_partition_bf has more than once bf(s),  and the bf indexed at 0  is just empty while bf(s) indexed at other slots are all not empty. These non-empty bf(s) can be used to filter data, is it supported?

Reply:
> when _hash_partition_bf has more than once bf(s), and the bf indexed at 0 is just empty while bf(s) indexed at other slots are all not empty. These non-empty bf(s) can be used to filter data, is it supported?

It's not supported now, clear_bf will clear all partitions' bf.  because of this, we can use _hash_partition_bf[0] to check if bf can be used.  If we want to support some partitions' bf to take effect, we may need more additional changes. I think we can optimize it later.

Reply:
I see,  may be we need refactor runtime filter calsses before other optimization.